lib/gemstash/cli/setup.rb

```diff
@@ -77,27 +77,27 @@ def ask_memcached_details
 
       def ask_database
         say_current_config(:db_adapter, "Current database adapter")
-        options = %w(sqlite3 postgres)
+        options = %w(sqlite3 postgres mysql)
         database = nil
 
         until database
-          database = @cli.ask "What database adapter? [SQLITE3, postgres]"
+          database = @cli.ask "What database adapter? [SQLITE3, postgres, mysql]"
           database = database.downcase
           database = "sqlite3" if database.empty?
           database = nil unless options.include?(database)
         end
 
         @config[:db_adapter] = database
-        ask_postgres_details if database == "postgres"
+        ask_database_details(database) unless database == "sqlite3"
       end
 
-      def ask_postgres_details
+      def ask_database_details(database = "postgres")
```

Comment:
Not really, it was just left there in the process of refactoring this as a safety measure.

I'll drop it.